luaui/Widgets/gui_point_tracker.lua

```diff
@@ -96,17 +99,23 @@ void main()
   vec4 worldPosInCamSpace;
 
   float viewratio = 1.0;
-   if (isMiniMap > 0.5) {
-    worldPosInCamSpace  = mmDrawViewProj * vec4(worldposradius.xyz, 1.0);
+  if (isMiniMap > 0.5) {
+    if (cameraFlipped > 0.5) {
+      worldPosInCamSpace  = mmDrawViewProj * vec4(mapSize.x - worldposradius.x, worldposradius.y, mapSize.y - worldposradius.z, 1.0);
```

Comment:
i presume `mapSize.y` actually stands for the Z coordinate? otherwise `mapSize.y - worldpos.z` looks suspicious

Reply:
Y in map coordinates is Z in gl coordinates